woocommerce/woocommerce
# Fix example extensions and move them to `create-extension` script

### All Submissions:






### Changes proposed in this Pull Request:



Closes # .






### How to test the changes in this Pull Request:



1. Load this branch and run `pnpm i` and then `pnpm run create-extension` and run `pnpm i && pnpm run start` on the newly created directory, this should all work correctly.
2. You may also want to add the plugin to a Wordpress site to see if it doesn't break anything.
3. Now run `pnpm run create-extension --example <example name>` -> `pnpm run create-extension --example add-task` for each example listed in the [README](https://github.com/woocommerce/woocommerce/blob/5bc6520e7b43a85a5c3bd2362891d775f289672d/tools/create-extension/README.md).
4. Run `pnpm i && pnpm run build` for each generated example and enable the plugin in Wordpress and make sure it works correctly.



### Other information:




### FOR PR REVIEWER ONLY:
